checker/src/org/checkerframework/checker/index/upperbound/OffsetEquation.java

```diff
@@ -387,7 +387,7 @@ private static int parseInt(String intLiteral) {
         if (intLiteral.isEmpty()) {
             return 0;
         }
-        return Integer.valueOf(intLiteral);
+        return Long.valueOf(intLiteral).intValue();
```

Comment:
To my surprise, no. I thought I did (i.e. there was a test that was failing at some point) but I can't reproduce it. I'll remove this code and see if the travis build fails.